src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,105 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidUserException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
 public class HelloWorldTest {
 
+    private RegistrationService registrationService;
+    private StorageDao storageDao;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        // Clear storage before each test
+        core.basesyntax.db.Storage.people.clear();
+    }
+
+    @Test
+    public void register_validUser_userRegistered() {
```

Comment:
The test case `register_validUser_userRegistered` does not verify if the user was actually added to the storage. An assertion to check the presence of the user in the storage is needed to ensure the registration was successful.